### PR TITLE
add bash script to zip and upload iteration to s3 #1627

### PR DESCRIPTION
**Pull request by twoface88**
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/1628)

#1627

### CODE REVIEW

**Comment by twoface88** [comment]
Should be called from root directory like this

`./src/main/bash/zipRootAndLastIterationAndUploadToS3.sh "output/sf-light/sf-light__2019-04-04_20-10-26" 0`

File: src/main/bash/zipRootAndLastIterationAndUploadToS3.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [ "$#" -ne 2 ]; then
+    echo "Illegal number of parameters";
+    exit 1;
+fi
+
+zip -r beam.zip . -x "output/*"
```
Comment (wrashid) [inline src/main/bash/zipRootAndLastIterationAndUploadToS3.sh]: keep name of folder provided in input and take that as name for zip file name

**Comment by twoface88** [comment]
unzip -l sf-light__2019-04-04_20-10-26.zip "*/"

```
        0  03-26-2019 19:07   src/main/scala/beam/agentsim/infrastructure/
        0  03-26-2019 19:07   src/main/scala/beam/analysis/
        0  03-30-2019 10:44   src/main/scala/beam/calibration/
        0  03-26-2019 19:07   src/main/scala/beam/calibration/impl/
        0  03-26-2019 19:07   src/main/scala/beam/calibration/impl/example/
        0  03-30-2019 10:44   src/main/scala/beam/calibration/utils/
        0  03-26-2019 19:07   src/main/scala/beam/calibration/api/
        0  04-04-2019 19:54   src/main/scala/beam/utils/
        0  03-26-2019 19:07   src/main/scala/beam/utils/plan/
        0  03-26-2019 19:07   src/main/scala/beam/utils/plan/sampling/
        0  04-04-2019 19:54   src/main/scala/beam/utils/scenario/
        0  04-04-2019 19:54   src/main/scala/beam/utils/scenario/matsim/
        0  04-04-2019 22:32   src/main/scala/beam/utils/scenario/urbansim/
        0  03-26-2019 19:07   src/main/scala/beam/utils/scripts/
        0  03-26-2019 19:07   src/main/scala/beam/utils/matsim_conversion/
        0  03-26-2019 19:07   src/main/scala/beam/utils/logging/
        0  03-26-2019 19:07   src/main/scala/beam/utils/reflection/
        0  03-26-2019 19:07   src/main/scala/beam/docs/
        0  03-26-2019 19:07   src/main/scala/beam/replanning/
        0  03-26-2019 19:07   src/main/scala/beam/replanning/utilitybased/
        0  03-26-2019 19:07   src/main/scala/beam/scoring/
        0  03-26-2019 19:07   src/main/scala/beam/experiment/
        0  04-04-2019 19:56   src/main/scala/beam/sim/
        0  03-26-2019 19:07   src/main/scala/beam/sim/metrics/
        0  04-04-2019 22:32   src/main/scala/beam/sim/config/
        0  03-26-2019 19:07   src/main/scala/beam/sim/population/
        0  03-26-2019 19:07   src/main/scala/beam/sim/common/
        0  03-27-2019 21:33   src/main/scala/beam/sim/vehicles/
        0  03-26-2019 19:07   src/main/scala/beam/sim/monitoring/
        0  03-26-2019 19:07   src/main/scala/beam/sim/vehiclesharing/
        0  03-26-2019 19:07   src/main/scala/beam/sim/modules/
        0  03-26-2019 19:07   src/main/scala/beam/physsim/
        0  03-26-2019 19:07   src/main/scala/beam/physsim/jdeqsim/
        0  03-26-2019 19:07   src/main/scala/beam/physsim/jdeqsim/cacc/
        0  04-04-2019 22:32   src/main/scala/beam/router/
        0  03-26-2019 19:07   src/main/scala/beam/router/osm/
        0  03-26-2019 19:07   src/main/scala/beam/router/gtfs/
        0  03-26-2019 19:07   src/main/scala/beam/router/model/
        0  03-26-2019 19:07   src/main/scala/beam/router/r5/
        0  04-04-2019 20:13   output/sf-light/sf-light__2019-04-04_20-10-26/ITERS/it.0/
        0  04-04-2019 20:12   output/sf-light/sf-light__2019-04-04_20-10-26/ITERS/it.0/graphs/
        0  04-04-2019 20:12   output/sf-light/sf-light__2019-04-04_20-10-26/ITERS/it.0/graphs/div/
        0  04-04-2019 20:12   output/sf-light/sf-light__2019-04-04_20-10-26/ITERS/it.0/graphs/png/
        0  04-04-2019 20:12   output/sf-light/sf-light__2019-04-04_20-10-26/ITERS/it.0/tripHistogram/
---------                     -------
        0                     1076 file
```

**Comment by twoface88** [comment]
`./src/main/bash/zipRootAndLastIterationAndUploadToS3.sh "sf-light__2019-04-04_20-10-26" 0`

**Pull request merged**